style.css

```diff
@@ -0,0 +1,718 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* ------------------navbar section------------------- */
+.nav-container {
+  display: flex;
+  width: 1440px;
+  height: 72px;
+  background-color: #3e3c3c;
+  align-items: center;
+}
+
+.nav-container a {
+  display: flex;
+  align-items: center;
+  margin-left: 20px;
+  flex: 1;
+  height: 100%;
+  color: white;
+  text-decoration: none;
+  list-style: none;
+}
+
+.nav-container .items-menu {
+  display: flex;
+  flex: 2;
+  justify-content: center;
+  padding-left: 200px;
+  height: 100%;
+  align-items: center;
+}
+
+.nav-container .items-menu a {
+  margin-right: 40px;
+  color: white;
+  list-style: none;
+  text-decoration: none;
+  font-size: 18px;
+  height: 100%;
+  padding-left: 10px;
+  padding-right: 10px;
+  transition: ease-in-out 5s;
+}
+
+.nav-container .items-menu a:hover {
+  background-color: #615f5f;
+}
+
+.nav-container a li .fa-bars {
+  display: none;
+}
+
+/* ---------------------------------------hero section------------------------------------ */
+.hero-section .hero-row {
+  display: flex;
+  background-color: #1c1a19;
+  width: 1440px;
+  height: 500px;
+}
+
+.hero-left {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-left: 30px;
+  margin-top: 100px;
+  flex: 0.2;
+  height: 100px;
+  transition: transform 5s;
+  order: 1;
+}
+
+.hero-left a i {
+  background: #1c1a19;
+  color: #cfcfcf;
+}
+
+.hero-left a i:hover {
+  color: brown;
+  border-radius: 50px;
+}
+
+.hero-right {
+  display: flex;
+  flex-direction: column;
+  margin-left: 40px;
+  margin-top: 60px;
+  flex: 0.5;
+  height: 300px;
+  order: 2;
+}
+
+.hero-right h1 {
+  font-size: 26px;
+  color: rgb(155, 116, 19);
+}
+
+.hero-right h5 {
+  color: rgb(204, 194, 194);
+  font-size: 18px;
+}
+
+.hero-right p {
+  margin-top: 20px;
+  color: rgb(204, 194, 194);
+  font-size: 14px;
+}
+
```

Comment:
- [ ] Kindly make sure to use the same typography styles in order to meet the provided Figma design as much as possible.

   - ### YOUR DESIGN
   
       ![image](https://user-images.githubusercontent.com/87197412/212619738-9e7b3304-c78d-4fe5-a78d-f9388eb061e1.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/212619840-dc02fe5c-1c90-4402-bba2-4c345f8d9236.png)